suse_sphinx_theme/css.html

```diff
@@ -5,7 +5,8 @@
 <link href="{{pathto('_static/css/native.css', 1)}}" rel="stylesheet">
 
 <!-- Fonts -->
-<link href="{{pathto('_static/css/font-awesome.min.css', 1)}}" rel="stylesheet">
+<!--<link href="{{pathto('_static/css/font-awesome.min.css', 1)}}" rel="stylesheet">-->
```

Comment:
my concern was that if the upstream theme changes, it might be easier to rebase this with that present, but thats probably not a great reason, I'll yoink it out